assets/css/v2/pre_fare/top_level_switch.scss

```diff
@@ -0,0 +1,27 @@
+.pre-fare-screen-viewport {
+  width: 1080px;
```

Comment:
Doesn't have to be here, but I might rename this file `pre-fare-dual-screen.scss`